### PR TITLE
feat: v0.3.0 adapted to latest changes in witnet-ethereum-bridge

### PR DESCRIPTION
**Pull request by aesedepece**
feat: v0.3.0 adapted to latest changes in witnet-ethereum-bridge

Supersedes #23

### CODE REVIEW

File: scripts/postinstall.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ var os = require('os');
 
 switch (os.type()) {
   case "Windows_NT":
-    exec("mkdir build\\contracts\\ && copy node_modules\\witnet-ethereum-bridge\\build\\contracts\\*.json build\\contracts")
+    exec("mkdir build\\contracts\\ && copy node_modules\\witnet-ethereum-bridge\\build\\**\\**\\*.json build\\contracts")
```
Comment (tmpolaczyk) [inline scripts/postinstall.js:9]: Does `**\\**` work on windows?

Comment (aesedepece) [reply scripts/postinstall.js:9]: @guidiaz will confirm :rofl:

Comment (aesedepece) [reply scripts/postinstall.js:9]: My assumption is that this path will not be expanded by Windows but rather by `mkdir` itself :shrug:

**Comment by aesedepece** [comment]
@tmpolaczyk green light to this and therefore https://github.com/witnet/witnet-requests-js/pull/41?

**Comment by tmpolaczyk** [comment]
I'm getting a strange error when running `yarn test` or `yarn migrate`:

```
1_witnet_core.js
================

/home/tomasz/projects/truffle-box/node_modules/truffle/build/459.bundled.js:26523
        throw new Error("Could not find artifacts for " + import_path + " from any sources");
        ^

Error: Could not find artifacts for WitnetParserLib from any sources
    at Resolver.require (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/resolver/dist/lib/resolver.js:61:1)
    at Object.require (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/migrate/index.js:172:1)
    at ResolverIntercept.require (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/migrate/ResolverIntercept.js:22:1)
    at /home/tomasz/projects/truffle-box/migrations/1_witnet_core.js:5:35
    at Script.runInContext (vm.js:130:18)
    at Script.runInNewContext (vm.js:135:17)
    at Object.file (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/require/require.js:94:1)
    at Migration._load (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/migrate/Migration.js:44:1)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at Migration.run (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/migrate/Migration.js:217:1)
    at Object.runMigrations (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/migrate/index.js:150:1)
    at Object.runFrom (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/migrate/index.js:110:1)
    at Object.run (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/migrate/index.js:87:1)
    at runMigrations (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/core/lib/commands/migrate.js:258:1)
    at Object.run (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/core/lib/commands/migrate.js:223:1)
    at Command.run (/home/tomasz/projects/truffle-box/node_modules/truffle/build/webpack:/packages/core/lib/command.js:172:1)
Truffle v5.4.7 (core: 5.4.7)
Node v12.21.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by aesedepece** [comment]
Interesting, it used to work for me until yesterday but now I'm getting that same error. I'll investigate. Thanks!

**Comment by aesedepece** [comment]
@tmpolaczyk It should be fixed now. The first line here was missing:
https://github.com/witnet/truffle-box/blob/68b35561464ede91e558c7a42c6b0110c29697f3/scripts/postinstall.js#L12-L13

**Review by tmpolaczyk — APPROVED** [review]

**Pull request merged**